DISPATCH_QUEUE_CONCURRENT flagged as undefined behavior

```
error[E0080]: this static likely exhibits undefined behavior
   --> /Users/mehcode/.cargo/registry/src/github.com-1ecc6299db9ec823/dispatch-0.1.3/src/ffi.rs:138:1
    |
138 | pub static DISPATCH_QUEUE_CONCURRENT: &'static dispatch_object_s = unsafe { &_dispatch_queue_attr_concurrent };
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ type validation failed: encountered a value of an uninhabited type at .<deref>
    |
    = note: The rules on what exactly is undefined behavior aren't clear, so this check might be overzealous. Please open an issue on the rust compiler repository if you believe it should not be considered undefined behavior
```

---

```
❯ rustc --version
rustc 1.31.0-nightly (424a749a0 2018-10-21)
```

--- a/src/ffi.rs
+++ b/src/ffi.rs
@@ -3,7 +3,8 @@
 
 use std::os::raw::{c_char, c_long, c_ulong, c_void};
 
-pub enum dispatch_object_s { }
+#[repr(C)]
+pub struct dispatch_object_s { _private: [u8; 0] }
 
 // dispatch_block_t
 pub type dispatch_function_t = extern fn(*mut c_void);

